tm2py/components/network/transit/transit.py

```diff
@@ -1108,9 +1113,10 @@ def run_skims(self,
                 num_processors=num_processors,
             )
 
-            walk_perception_factor = self.controller.config.transit.get("walk_perception_factor", 2)
             drive_perception_factor = self.controller.config.transit.get("drive_perception_factor", 2)
-            # divide drive and walk time by mode specific perception factor to get the actual time
+            walk_speed = 3.0
```

Comment:
Can we expose this in the model_config.toml?

Reply:
added to model_config.toml in https://github.com/camsys/tm2py/pull/53/commits/cfce21695d62ed72e73dd31a858b23d9291cda59